docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-is-used-on-the-project-level.md

```diff
@@ -323,21 +345,27 @@ class CustomerAccessEntityManager extends SprykerCustomerAccessEntityManager imp
 }
 ```
 
-Example of related error in the Evaluator output:
+### Resolving the error
 
-```bash
-------------------------------------------------------------------------------------------------------------------------
-Please avoid usage of Spryker\...\CustomerAccessEntityManager::createCustomerAccessMapper(...) in Pyz/Zed/CustomerAccess/Persistence/CustomerAccessEntityManager
-------------------------------------------------------------------------------------------------------------------------
-```
+To resolve the error provided in the example, try the following in the provided order:
```

Comment:
Maybe we can create one section "resolving error" instead of copy-paste the same text?

Reply:
As I understood, last time @andriitserkovnyi was against the references on some other blocks, so I copied it for every problem.

@andriitserkovnyi , what do you think about that?